content/v3/oauth.md

```diff
@@ -14,7 +14,7 @@ be limited to specific types of data, and can be revoked by users at any
 time.
 
 All developers need to [register their
-application](https://github.com/settings/applications/new) before getting
+application](https://github.com/settings/applications/new) or [register a personal access token](https://github.com/settings/tokens/new) before getting
```

Comment:
I think this adds some confusion because personal access tokens are unrelated to the web flow below. 

@stevemao How about adding another sentence to the preceding paragraph. Something like:

> This is preferred over Basic Authentication because tokens can be limited to specific types of data, and can be revoked by users at any time. **You may create a [personal access token][] for your own use or implement the web flow below to allow other users to authorize your application.**

[personal access token]: https://github.com/blog/1509-personal-api-tokens

/cc @gjtorikian @keavy